### DISCUSSION TITLE
laravel/prompts

### DISCUSSION THREAD

**Discussion by batrox** [Power-Components/livewire-powergrid#1725] [Q&A]
Hi,

I installed powergrid, thank you for your work !

I had to downgrade laravel/prompts to 0.1.25 

Why don't you allowed 0.2.X and 0.3.X ?

In every package using prompts, they allow ^0.1.25|^0.2|^0.3

Cheers

**Comment by ismaildasci** [comment]
Hey! 👋

Thanks for using PowerGrid!

The restriction on `laravel/prompts` to version `0.1.25` might be due to compatibility issues or breaking changes introduced in `0.2.X` and `0.3.X`. Some packages may set strict version constraints to ensure stability, especially if newer versions have API changes or deprecated methods.

Allowing versions like `^0.1.25|^0.2|^0.3` could potentially introduce issues if PowerGrid relies on specific behaviors in `0.1.X`. However, it’s worth checking if there’s an upcoming update or a planned change to support these versions.

You might want to check with the maintainers if there’s an active branch or release that expands support to `0.2.X` and `0.3.X`, or if there are specific blockers preventing compatibility with these versions.

Cheers! 😊

**Comment by luanfreitasdev** [comment]
@ismaildasci, I agree with you, however, currently, any version one is supported because a stable version (1.x) has not been defined yet. I'll keep `"*"` for now.

Thank you
